app/controllers/api/v2/rules_of_origin_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
 
         @serializer = Api::V2::RulesOfOrigin::SchemeSerializer.new(
           presented_schemes,
-          include: %i[links proofs rules],
+          include: %i[links proofs rules articles],
```

Comment:
I debated whether to make the inclusion of articles conditionalized on a query param but figured it wasn't worth the added complexity since they're all pretty short markdown documents